lib/action_controller/serialization.rb

```diff
@@ -31,6 +31,7 @@ def get_serializer(resource, options = {})
         serializable_resource.serialization_scope_name = _serialization_scope
         begin
           serializable_resource.adapter
+          serializable_resource
```

Comment:
The main purpose of this is to the instrumentation be called on the Rails controller, so if I remove, [this test](https://github.com/maurogeorge/active_model_serializers/commit/981c2293f4ca46daf1ebcd51468497611a389687#diff-f290b93cc1e01ae4a2c31174202b50eeR424) will be broken.

As you can see [here](https://github.com/maurogeorge/active_model_serializers/blob/2c4a028bed946d10b2972c975ef9c2501ef43c25/lib/active_model/serializable_resource.rb#L14-L30) the instrumentation is made on the `ActiveModel::SerializableResource` what [I think is the canonical way to use a serializer](https://github.com/maurogeorge/active_model_serializers/tree/2c4a028bed946d10b2972c975ef9c2501ef43c25#using-a-serializer-without-render) and as you can see we simply delegate the method to the adapter. If we call the adapter directly over use the `ActiveModel::SerializableResource` we will need to change all adapters to call instrumentation on they methods `serializable_hash`, `as_json` and `to_json`. 

Keeping the `ActiveModel::SerializableResource` we define this once and all adapter, including new ones, will have instrumentation.

Let me know if you have some doubt.